src/assets/scripts/client/aircraft/FlightManagementSystem/LegModel.js

```diff
@@ -361,17 +361,11 @@ export default class LegModel {
      *
      * @for LegModel
      * @method _buildWaypointForHoldingPatternAtPosition
-     * @param routeString {string}
      * @param holdWaypointProps {object}
      * @return {array<WaypointModel>}
      */
-    _buildWaypointForHoldingPatternAtPosition(routeString, holdWaypointProps) {
-        const holdWaypointPropsWithPositionModel = this._buildHoldAtPositionWaypointProps(holdWaypointProps);
-        const waypointModel = new WaypointModel(holdWaypointPropsWithPositionModel);
-
-        return [
-            waypointModel
-        ];
+    _buildWaypointForHoldingPatternAtPosition(holdWaypointProps) {
+        return [new WaypointModel(holdWaypointProps)];
```

Comment:
could you reformat this to be like lines 370 - 374?  

whenever we create a new instance, with _very few_ exceptions, it should first be set to a variable or class property.